packages/design-system/src/components/Accordion/Primitive/CollapsiblePanelHeader.tsx

```diff
@@ -22,7 +22,7 @@ export type CollapsiblePanelHeaderPropsType = {
 	title?: ReactChild;
 	status?: keyof typeof variants;
 	metadata?: ReactChild[];
-	action?: PanelHeaderAction;
+	actions?: [PanelHeaderAction, PanelHeaderAction?];
```

Comment:
If we allow any number of actions
```suggestion
	actions?: PanelHeaderAction[];
```